packages/host/android/CMakeLists.txt

```
@@ -20,6 +20,8 @@ add_library(node-api-host SHARED
   ../cpp/Logger.cpp
   ../cpp/CxxNodeApiHostModule.cpp
   ../cpp/WeakNodeApiInjector.cpp
+  ../cpp/node_api_impl.cpp
+  ../cpp/node_api_impl.hpp
```

Any reasons we shouldn't follow the existing naming scheme here? Also, I wonder if it makes sense to reference the fact that this is intended for the implementation of the "runtime" part of Node-API?

```suggestion
  ../cpp/RuntimeNodeApi.cpp
  ../cpp/RuntimeNodeApi.hpp
```

---

It definitely makes sense to follow existing schema, initially I wanted to follow more node-ish naming convention and put files in separate directory, but if finally they ended up there I will adjust the name accordingly (including `Runtime` part).